### PR TITLE
fix correction link browser

### PR DESCRIPTION
**Pull request by Monia-LAOUFI**
--> link to Trello: https://trello.com/c/nLom4GOl

# Detailed purpose of the PR
. This PR aims to make the link of the card clickable. 
. This link opens to a modal 

# Testing strategy
- [x] I ensured my code is tested: unit 

# Result

[link-card-browser.gif.zip](https://github.com/CoorpAcademy/mobile/files/4362115/link-card-browser.gif.zip)

### CODE REVIEW

**Comment by Monia-LAOUFI** [comment]
[Mobile App: html links unsupported in interquestion popins](https://trello.com/c/nLom4GOl/1469-mobile-app-html-links-unsupported-in-interquestion-popins)

File: src/__fixtures__/discipline-bundle/fix-correction-link-browser.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    })
+  },
+  exitNodes: {
+    [failureExitNode.ref]: failureExitNode,
+    [successExitNode.ref]: successExitNode
+  },
+  chapterRules: {}
+};
+
+export default bundledDiscipline;
```
Comment (VincentCATILLON) [inline src/__fixtures__/discipline-bundle/fix-correction-link-browser.js]: imo this file is not useful as you change the default fixture file used by all bundles :)

File: src/__fixtures__/discipline-bundle/index.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import template from './template';
 import qcmDrag from './with-qcm-drag';
 import slider from './slider';
 import locked from './locked';
+import fixLink from './fix-correction-link-browser';
```
Comment (VincentCATILLON) [inline src/__fixtures__/discipline-bundle/index.js]: same than above

File: src/__fixtures__/discipline-bundle/index.js [hunk 3]
```diff
@@ -24,7 +25,8 @@ const items = [
   qcmDrag,
   slider,
   withBasicQuestion,
-  locked
+  locked,
+  fixLink
```
Comment (VincentCATILLON) [inline src/__fixtures__/discipline-bundle/index.js]: same than above

File: src/__fixtures__/slides.js [hunk 4]
```diff
@@ -23,9 +23,9 @@ export const createSlide = ({
   universalRef: ref,
   clue: clue === null ? undefined : clue || 'Une question de gestion.',
   klf:
-    'Open Compute Project permet de diffuser les solutions les plus efficaces de gestion énergétique dans les data centers et ainsi de réduire la consommation des grandes entreprises.',
+    "Follow this <a href='https://coorpacademy.com'>linkCoorp</a> Open Compute Project permet de diffuser les solutions les plus efficaces de gestion énergétique dans les data centers et ainsi de réduire la consommation des grandes entreprises.",
```
Comment (VincentCATILLON) [inline src/__fixtures__/slides.js]: 👍

File: src/screens/correction.js [hunk 5]
```diff
@@ -94,6 +94,13 @@ class CorrectionScreen extends React.PureComponent<Props> {
 
   handleVideoPlay = () => this.props.play();
 
+  handleLinkPress = (url: string): void => {
+    const {
+      navigation: {navigate}
+    } = this.props;
+    navigate('BrowserModal', {url});
```
Comment (VincentCATILLON) [inline src/screens/correction.js]: can you type the params please ? as [here](https://github.com/CoorpAcademy/mobile/blob/9aa9b476f68b77e29bb1982962d85a8049ba4b04/src/screens/context.js#L35)

**Pull request merged**